### PR TITLE
Sub menus #672

### PR DESCRIPTION
**Pull request by borislavstoychev**
## Screenshots:

| Before              | After              |
| ------------------- | ------------------ |
| [before screenshot] | [after screenshot] |

## Testing

**New environment variables**:

- `env var` : env var details

**New dependencies**:

- `dependency` : dependency details

**New dev dependencies**:

- `dependency` : dependency details

### CODE REVIEW

**Comment by borislavstoychev** [comment]
image
image
image
image

**Comment by borislavstoychev** [comment]
Screenshot from 2022-04-07 21-12-49
I'm not sure about the settings button below. Should be there or not?

**Comment by kachar** [comment]
> I'm not sure about the settings button below. Should be there or not?

Let's leave the settings button there for now, it might receive greater purpose at some point.

File: src/components/admin/navigation/AdminLayout.tsx [hunk 1]
```diff
@@ -190,16 +197,17 @@ export default function AdminLayout({ children }: Props) {
...
-              icon={<Icon />}
-              label={label}
-              onClick={() => router.push(href)}
+              menu={mainMenu[index].menu}
+              icon={mainMenu[index].icon}
+              submenu={submenu}
             />
           ))}
           <CustomListItem icon={open ? <MenuOpen /> : <ChevronRight />} onClick={toggleMenu} />
+          {/* I'm not sure if I need to delete this settings button */}
```
Comment (kachar) [inline src/components/admin/navigation/AdminLayout.tsx:210]:
```suggestion
```

**Pull request merged**